src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -321,8 +321,12 @@ private async Task<long> CreateGlobalStateTableAsync(SqlConnection connection, S
                         UserFunctionID char(16) NOT NULL,
                         UserTableID int NOT NULL,
                         LastSyncVersion bigint NOT NULL,
+                        LastAccessTime Datetime NULL,
```

Comment:
Let's change this to NOT NULL and default to GETDATE. That makes it simpler, and there's no reason I can think of to have this actually be NULL